website/blog/2024-10-28-release-1.14.md

```diff
@@ -0,0 +1,220 @@
+---
+title: Podman Desktop 1.14 Release
+description: Podman Desktop 1.14 has been released!
+slug: podman-desktop-release-1.14
+authors: [gastoner]
+tags: [podman-desktop, release, kubernetes]
+hide_table_of_contents: false
+<!-- This image link is used for social media previews / thumbnails. Release images are available: https://github.com/containers/podman-desktop-internal/tree/main/release-images -->
+image: /blog/img/podman-desktop-release-1.14/banner.png
+---
+
+
+<!-- import ReactPlayer from 'react-player' -->
+
+Podman Desktop 1.14 Release! 🎉
+
+![podman-desktop-hero-1.14](img/podman-desktop-release-1.14/banner.png)
+
+Podman Desktop 1.14 is now available. [Click here to download it](/downloads)!
+
+This release includes: 
+
+- **Kubernetes improvements**: UI changes that will make life with working with kuberneres easier.
+- **Port forwarding**: This new feature allows to configure port forwarding in kubernetes environment.
```

Comment:
'allows to configure' isn't very clear, maybe 'allows users to configure' or 'allows users to set up'